cmd/admin-rpc-client.go

```diff
@@ -82,6 +82,16 @@ func (rpcClient *AdminRPCClient) WriteTmpConfig(tmpFileName string, configBytes
 	return err
 }
 
+// BackupConfig - saves a copy of the config file
+func (rpcClient *AdminRPCClient) BackupConfig() error {
+	args := BackupConfigArgs{}
+	reply := VoidReply{}
+
+	err := rpcClient.Call(adminServiceName+".BackupConfig", &args, &reply)
+	logger.LogIf(context.Background(), err)
```

Comment:
This function can receive a context for the logger.

Reply:
> This function can receive a context for the logger.

Can you do that in a separate PR ? because the same thing needs to be done to AdminRPCConfig

Reply:
ok. In general, the idea is that when we have a high level handler function. We will create a context there and try to pass the context to the functions underneath